src/scripts/achievements/ProteinObtainRequirement.ts

```diff
@@ -0,0 +1,15 @@
+///<reference path="AchievementRequirement.ts"/>
+
+class ProteinObtainRequirement extends AchievementRequirement {
+    constructor(value: number, option: GameConstants.AchievementOption = GameConstants.AchievementOption.more) {
+        super(value, option, GameConstants.AchievementType['Protein']);
+    }
+
+    public getProgress() {
+        return Math.min(App.game.statistics.totalProteinsObtained(), this.requiredValue);
+    }
+
+    public hint(): string {
+        return `${this.requiredValue} proteins need to be obtained in total. Used proteins stay counted.`;
```

Comment:
I think we should keep the hint similar to the Diamonds/Money etc hints

Reply:
If the other currency chieves count total cumulative amounts instead of what the player has at the moment, then keeping the hints the same makes sense. No problemo, I’ll get on those tasks soon.